src/main/java/org/codehaus/plexus/util/FileUtils.java

```diff
@@ -374,11 +375,11 @@ private static InputStreamReader getInputStreamReader( File file, String encodin
     {
         if ( encoding != null )
         {
-            return new InputStreamReader( new FileInputStream( file ), encoding );
+            return new InputStreamReader( Files.newInputStream( file.toPath() ), encoding );
```

Comment:
Why not `Files#newBufferedReader()`?

Reply:
Because that is `@since 1.8`, while Plexus still is limited to 1.7. Indeed I have already prepared the needed changes for 1.8, 1.9 and 1.10 in separate branches, so if Plexus steps up the minimum level, I can instantly send a PR for that.

Reply:
Thanks!